src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -390,8 +354,9 @@ private void persistCredentials() {
 
     for (Account account : accountRoster.getAccounts()) {
       OAuthData oAuthData = new OAuthData(account.getAccessToken(), account.getRefreshToken(),
-          account.getEmail(), oAuthScopes, account.getAccessTokenExpiryTime());
+          account.getEmail(), account.getName(), account.getAvatarUrl(),
+          oAuthScopes, account.getAccessTokenExpiryTime());
       authDataStore.saveOAuthData(oAuthData);
```

Comment:
Seems `OAuthData` and `OAuthDataStore` are now saving far more than just OAuth data.